packages/server/api/routes/uidreview.router.js

```diff
@@ -0,0 +1,22 @@
+const express = require('express');
+
+const router = express.Router({ mergeParams: true });
+// controllers
+const uidreviewController = require('../controllers/uidreview.controller');
+
+router.get('/:id/:uid', (req, res) => {
+  uidreviewController
```

Comment:
Not sure if a new router is needed. Could you not put this inside moviecontroller or review controller?
Route should represent some sort of business object or entity

Reply:
I have change it to the review controller